cv_next/app/hall/page.tsx

```diff
@@ -1,54 +1,81 @@
+import Image from "next/image";
 import { ScrollToTop } from "@/components/ui/scrollToTop";
 import { External_Credits as contributors } from "@/lib/definitions";
 
 const hall: React.FC = () => {
   return (
-    <div className="relative mx-auto h-full w-[700px] max-w-full xl:w-[1400px]">
-      <ScrollToTop />
-      <div className="p-1 text-center">
-        <h1 className="mb-5 inline-flex items-center pt-3 text-2xl font-bold text-gray-900 dark:text-white sm:text-2xl md:text-3xl lg:text-4xl xl:text-5xl">
-          A Huge Thank You to Our Contributors
-        </h1>
-        <h3 className="pb-8 text-base font-light text-muted-foreground lg:text-lg">
-          Want to also be in this list? Contact us in the community chat and win
-          eternal glory!
-        </h3>
-        <ul>
-          <div className="justify-center xl:flex">
-            <div
-              id="grid-container"
-              className="w450:grid-cols-1 w450:gap-7 grid w-full grid-cols-2 gap-10 md:grid-cols-3 xl:w-1/2"
-            >
+    <>
+      <link
+        rel="stylesheet"
+        href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.7.0/css/all.min.css"
+      />
+      <title>Hall of fame</title>
+      {/* stars background */}
+      <ul className="crowns absolute left-0 top-0 z-[-1] h-[100%] w-[100%] overflow-hidden">
+        {Array.from({ length: 15 }).map((_, index) => (
+          <li key={index}></li>
+        ))}
+      </ul>
+      <div className="relative mx-auto h-full w-[700px] max-w-full xl:w-[1400px]">
+        <ScrollToTop />
+        <div className="select-none p-5 text-center">
+          <h1 className="hall-title mb-5 inline-flex items-center pt-3 text-2xl font-bold text-gray-900 dark:text-white sm:text-2xl md:text-3xl lg:text-4xl xl:text-5xl">
+            HALL OF FAME
+          </h1>
+          <h3 className="pb8 font-bold lg:text-lg">
+            Awesome people who contributed to the project
+          </h3>
+          <h3 className="pb-8 text-base font-light text-muted-foreground lg:text-lg">
+            Want to also be in this list? Contact us in the community chat and
+            win eternal glory!
+          </h3>
+          <ul>
+            <div className="flex max-h-[80vh] w-full flex-row-reverse flex-wrap justify-center justify-center self-center overflow-y-auto overflow-x-hidden xl:flex">
+
               {contributors.map((contributor, index) => (
-                <a
+                // key = index for now, but its bad practice..
+                // https://robinpokorny.com/blog/index-as-a-key-is-an-anti-pattern/
```

Comment:
dangling comments

Reply:
or is it purposefully left here?
you don't have to keep the index as key..

Reply:
changed it to index by name